src/Raven.Server/ServerWide/Commands/ModifyDatabaseWatchersCommand.cs

```diff
@@ -20,15 +20,16 @@ public ModifyDatabaseWatchersCommand(string databaseName) : base(databaseName)
 
         public override string UpdateDatabaseRecord(DatabaseRecord record, long etag)
```

Comment:
return value was changed in this commit: https://github.com/ravendb/ravendb/commit/faf89d5c97158475dfc5789eed911be8dd2f1a7e
it believe that's the reason: https://github.com/ravendb/ravendb/blob/faf89d5c97158475dfc5789eed911be8dd2f1a7e/src/Raven.Server/ServerWide/ClusterStateMachine.cs#L480